src/lib/nodes/services.ts

```diff
@@ -1,7 +1,9 @@
 import { klyIndexer } from './kly-indexer'
 import { ethIndexer } from './eth-indexer'
+import { rateInfoClient } from './rate-info-service'
 
 export const services = {
   klyIndexer,
-  ethIndexer
+  ethIndexer,
+  'rates-info': rateInfoClient
```

Comment:
```suggestion
  ratesInfo: rateInfoClient
```

Better to follow same camelCase style